week2/5_String_Methods.ipynb

```diff
@@ -433,15 +417,16 @@
     "# נסו להכניס הרבה רווחים אחרי או לפני שם המשתמש\n",
     "username = input(\"Please enter your user: \")\n",
     "username = username.strip()\n",
-    "print(\"This string is: \" + username + \".\")"
+    "print(f\"This string is: {username}.\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "<p style=\"text-align: right; direction: rtl; float: right; clear: both;\">\n",
-    "    כשמביאים לה מחרוזת כארגומנט, היא תבצע את הפסאודוקוד הבא:\n",
+    "     כאשר מעבירים לה מחרוזת כארגומנט, היא תבצע את הפסאודוקוד הבא:<br>\n",
+    "    * <dfn>פסאודוקוד</dfn> הוא תיאור מצומצם וקריא לאלגוריתם במחשב אשר נראה כמו קוד אך בפועל אינו מייצג אף שפת תכנות.<br>\n",
```

Comment:
I think we should use "Algorithm" and create this explanation in the summary of week1.
This line can divert the students from the goal, and we want them 100% focused.
We should also create a new notebook in week1, which explains basic CS intro.